src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -187,53 +252,20 @@ export const SueReportLocation = ({
           locations={locations}
           mapCenterPosition={mapCenterPosition}
           mapStyle={styles.map}
-          onMyLocationButtonPress={() =>
-            Alert.alert(texts.sue.report.alerts.hint, texts.sue.report.alerts.myLocation, [
-              {
-                text: texts.sue.report.alerts.no
-              },
-              {
-                text: texts.sue.report.alerts.yes,
-                onPress: async () => {
-                  const location = position || lastKnownPosition;
-
-                  if (location) {
-                    setUpdatedRegion(true);
-                    setUpdateRegionFromImage(false);
-                    setSelectedPosition(location.coords);
-
-                    try {
-                      await handleGeocode(location.coords);
-                    } catch (error) {
-                      setSelectedPosition(undefined);
-                      Alert.alert(texts.sue.report.alerts.hint, error.message);
-                    }
-                  }
-                }
-              }
-            ])
-          }
-          onMapPress={async ({ nativeEvent }) => {
-            if (
-              nativeEvent.action !== 'marker-press' &&
-              nativeEvent.action !== 'callout-inside-press'
-            ) {
-              setUpdatedRegion(false);
-              setUpdateRegionFromImage(false);
-              setSelectedPosition(nativeEvent.coordinate);
-
-              try {
-                await handleGeocode(nativeEvent.coordinate);
-              } catch (error) {
-                setSelectedPosition(undefined);
-                Alert.alert(texts.sue.report.alerts.hint, error.message);
-              }
-            }
-          }}
+          onMyLocationButtonPress={() => onMyLocationButtonPress()}
```

Comment:
could be `onMyLocationButtonPress={onMyLocationButtonPress}` because there are not params passed, right?